gammapy/data/event_list.py

```diff
@@ -102,7 +102,13 @@ def read(cls, filename, **kwargs):
         table = Table.read(filename, **kwargs)
         return cls(table=table)
 
-    def write(self, filename, gti=None, overwrite=False, format='gadf'):
+    def to_table_hdu(self, name="EVENTS", format='gadf-dl3'):
+        '''
+        Convert event list to a `~astropy.io.fits.BinTableHDU`
```

Comment:
Please add Parameter section to docs and add a check for the value of `format`...